apps/advisor/pkg/app/checks/plugincheck/check.go

```diff
@@ -241,3 +261,24 @@ func (s *updateStep) isProvisioned(ctx context.Context, pluginID string) bool {
 	}
 	return slices.Contains(s.provisionedPlugins, pluginID)
 }
+
+// Temporary duplicated code until there is a common IsUpdatable function
```

Comment:
cc / @oshirohugo this should also go away with https://github.com/grafana/grafana/pull/104112